crates/goose-mcp/src/developer/mod.rs

```diff
@@ -152,13 +152,16 @@ impl DeveloperRouter {
                 If you need to run a long lived command, background it - e.g. `uvicorn main:app &` so that
                 this tool does not run indefinitely.
 
+                **Important**: Use ripgrep - `rg` - exclusively when you need to locate a file or a code reference,
+                other solutions may produce too large output because of hidden files! For example *do not* use `find` or `ls -r`
+                  - List files by name: `rg --files | rg <filename>`
+                  - List files that contain a regex: `rg '<regex>' -l`
+
```

Comment:
should we also update the windows instructions. or better, make them as much as possible the same

Reply:
yeah, maybe to increase maintainability extract the commonalities so we will keep them in sync better going forward?